query-creator/create-query/steps/firebase.js

```diff
@@ -79,7 +79,8 @@ const denormalizeActivity = (activity, denormalized) => {
     section.children.forEach(page => {
       page.children.forEach(question => {
         denormalized.questions[question.id] = {
-          prompt: question.prompt
+          prompt: question.prompt,
+          required: question.required ? question.required : false
```

Comment:
Here's the change to the `denormalizedResource` that I mentioned in my comment.  Not sure if this is safe.